src/components/Stepper/src/Stepper.vue

```diff
@@ -13,9 +13,30 @@
 		>
 			<minus :class="$s.Icon" />
 		</m-button>
-		<span :class="$s.Quantity">
-			{{ value }}
-		</span>
+		<div
+			:class="$s.Quantity"
+		>
+			<input
```

Comment:
We should run this functionality past design. 

I'm making some assumptions, but I think we'd get better styling / states / and interaction behavior by using our Maker input internally here.

Reply:
That was my initial approach, but based on the designs I've been given, it's not _currently_ possible with `MInput` in it's current state (eg too big, can't remove border _and_ background). I'll confirm with design if this is still the direction we want to go with.

Reply:
Just confirmed, design wants it (visually) how I've implemented.